css/style.css

```diff
@@ -198,7 +205,7 @@ header, .repl-container {
   }
 }
 
-.example-code::before {
+.example-input::before, .example-output::before, .code-block::before {
```

Comment:
I moved the pseudo selector up so "input" and "output" would be visible even when the code was scrolled horizontally.